k8s/orb.yaml

```
@@ -19,7 +19,7 @@ commands:
         type: string
         description: |
           Specify a list of container image updates in the space-delimited format.
-          (e.g. container1:latest nginx:1.9.1)
+          (e.g. ruby:latest nginx:1.9.1)
```

こんな感じでさらに明確にしてもいいかなって思いました
```suggestion
          (e.g. ruby:latest your_repository.example.com/foo:bar)
```